### DISCUSSION TITLE
Improve DX using pass-through for components with simple additional classes

### DISCUSSION THREAD

**Discussion by robomatic** [primefaces/community#839] [PrimeVue] [answered]
It would be great if we could simplify extending the primvue component pass-through. In particular when using primevue/tailwind. In the case when right now we have to write this to add a simple additional class:
```
<ConfirmDialog :pt="{ root: { class: 'min-w-[20vw]' } }" :pt-options="{ mergeProps: true }" />
```
If instead we could do something like this:
```
<ConfirmDialog :pt-extend="{ root: { class: 'min-w-[20vw]' } }" />
 ```
or
```
<ConfirmDialog :pt="{ extend: { root: { class: 'min-w-[20vw]' } } }" />
 ```

**Comment by tugcekucukoglu** [comment, marked as answer]
For your information, we're planning to deprecate the primevue/passthrough/tailwind and suggest using https://tailwind.primevue.org/overview/. So there is no need to use `ptOptions`

**Reply by robomatic** [reply]
I am currently transitioning from `primevue/passthrough/tailwind` and using `primevue/tailwind`, however there are still times when I want to customize a component. What's the recommended approach in that case?
